game.js

```diff
@@ -0,0 +1,532 @@
+        var x = document.getElementById("button2");
+        x.style.backgroundColor = "deepPink";
+    }
+    else {
+        var x = document.getElementById("button2");
+        x.style.backgroundColor = "#A9A9A9";
+    }
+}
+
+function hoverR() {
+    var x = document.getElementById("button3");
+    x.style.backgroundColor = "#C71585";
+}
+
+function notHoverR() {
+    var x = document.getElementById("button3");
+    x.style.backgroundColor = "deepPink";
+}
+
+function easyClicked() {
+    "use strict";
+    restartClicked();
+    var x = document.getElementById("button1"),
+        y;
+    x.innerHTML = "Easy";
+    y = document.getElementById("levels");
+    y.style.opacity = 0;
+    y.style.visibility = "hidden";
+    level = 1;
+    x = document.getElementById("button2");
+    x.style.backgroundColor = "#A9A9A9";
+    y = document.getElementById("button1");
+    y.style.backgroundColor = "deepPink";
+    if (human === 0) {
+        vsBotFirst();
+    }
+}
+
+function mediumClicked() {
+    "use strict";
+    restartClicked();
+    var x = document.getElementById("button1"),
+        y;
+    x.innerHTML = "Medium";
+    y = document.getElementById("levels");
+    y.style.opacity = 0;
+    y.style.visibility = "hidden";
+    level = 2;
+    x = document.getElementById("button2");
+    x.style.backgroundColor = "#A9A9A9";
+    y = document.getElementById("button1");
+    y.style.backgroundColor = "deepPink";
+    if (human === 0) {
+        vsBotFirst();
+    }
+}
+
+function hardClicked() {
+    "use strict";
+    restartClicked();
+    var x = document.getElementById("button1"),
+        y;
+    x.innerHTML = "Impossible";
+    y = document.getElementById("levels");
+    y.style.opacity = 0;
+    y.style.visibility = "hidden";
+    level = 3;
+    x = document.getElementById("button2");
+    x.style.backgroundColor = "#A9A9A9";
+    y = document.getElementById("button1");
+    y.style.backgroundColor = "deepPink";
+    if (human === 0) {
+        vsBotFirst();
+    }
+}
+
+function initVsPlayer() {
+    "use strict";
+    restartClicked();
+    level = 0;
+    var x = document.getElementById("button2"),
+        y = document.getElementById("button1");
+    x.style.backgroundColor = "deepPink";
+    y.style.backgroundColor = "#A9A9A9";
+    y.innerHTML = "Vs Bot";
+}
+
+function switchXO(i) {
+    "use strict";
+    if (moves > 0) {
+        return;
+    }
+    if (level <= 0) {
+        return;
+    }
+    var x, y;
+    if (i === 0) {
+        human = 0;
+        comp = 1;
+        x = document.getElementById("button-o");
+        y = document.getElementById("button-x");
+        x.style.backgroundColor = "deepPink";
+        y.style.backgroundColor = "#A9A9A9";
+
+        vsBotFirst();
+    } else {
+        human = 1;
+        comp = 0;
+        x = document.getElementById("button-o");
+        y = document.getElementById("button-x");
+        x.style.backgroundColor = "#A9A9A9";
+        y.style.backgroundColor = "deepPink";

```

Comment:
Your JS is really well-written, but there's a lot of interaction between JS and CSS, which is not the best way to write maintainable code. [This one's](https://stackoverflow.com/questions/22809850/best-practice-for-manipulating-css-from-javascript) a simple problem you may face with this approach.